### PR TITLE
Doc styling

### PR DESCRIPTION
**Pull request by asvetlov**
New documentation theme

#2520

### CODE REVIEW

**Comment by kxepal** [comment]
Sounds exciting. Do you have preview link?

**Comment by asvetlov** [comment]
https://www.dropbox.com/s/18vim055outms19/Screenshot%20from%202018-02-07%2018-06-29.png?dl=0

**Comment by kxepal** [comment]
Awesome!

**Comment by kxepal** [comment]
Though, I more like the old logo. It looks good on dark background and pretty warm in overall.

**Comment by asvetlov** [comment]
New logo is simpler, stricter and more printable (think about awesome t-shirt)

**Comment by asvetlov** [comment]
Should keep old logo in the repo though to don't dig into git history

File: docs/aiohttp-icon.svg [hunk 1]
```diff
@@ -1,487 +1 @@
...
...5,53.2l2.3,4.986a2.366,2.366,0,0,1,.333-.024,2.345,2.345,0,0,1,2.345,2.344,2.34,2.34,0,0,1-.8,1.76Zm9.036-11.612A34.334,34.334,0,0,0,2.232,37.679,34.384,34.384,0,0,0-5.593,15.811l-4.747,1.618a2.35,2.35,0,0,1,.065.551,2.345,2.345,0,0,1-2.345,2.345,2.346,2.346,0,0,1-2.142-1.389L-24.63,22.3a7.827,7.827,0,0,1,.283,2.091,7.81,7.81,0,0,1-3.47,6.5l8.969,15.391a3.9,3.9,0,0,1,1.537-.314A3.909,3.909,0,0,1-13.4,49.874q0,.146-.011.29Zm-27.778-.211,7.74-1.979a3.948,3.948,0,0,1-.033-.507,3.9,3.9,0,0,1,1.56-3.125l-8.967-15.387a7.783,7.783,0,0,1-3.538.844A7.781,7.781,0,0,1-36.3,31.026L-45,43.243a3.9,3.9,0,0,1,1.265,2.879,3.913,3.913,0,0,1-.154,1.088l10.937,4.7a2.341,2.341,0,0,1,1.88-.944A2.346,2.346,0,0,1-28.926,52.361Zm-22.513-7.172a3.911,3.911,0,0,1,3.8-2.975,3.891,3.891,0,0,1,1.882.482l8.7-12.213a7.8,7.8,0,0,1-2.92-6.093,7.787,7.787,0,0,1,1.936-5.15L-49.666,7.808A34.582,34.582,0,0,0-66.873,37.679a34.6,34.6,0,0,0,.47,5.7Zm26.5-23.778,9.969-3.4q0-.017,0-.034a2.346,2.346,0,0,1,2.345-2.345,2.342,2.342,0,0,1,1.878.94l4.5-1.534A34.485,34.485,0,0,0-32.32,3.127a34.325,34.325,0,0,0-16.5,4.2l11.43,11.244a7.787,7.787,0,0,1,5.222-2A7.82,7.82,0,0,1-24.934,21.411Z" transform="translate(433.82 73.82)"/></svg>
```
Comment (webknjaz) [inline docs/aiohttp-icon.svg]: Did it have to be one-liner?

Comment (asvetlov) [reply docs/aiohttp-icon.svg]: Sorry, I have no idea.
I did not edit the file manually :)

Comment (webknjaz) [reply docs/aiohttp-icon.svg]: It's much more usable in prettified form. Front-end dudes are able to work with that, so I'd keep it readable.

**Comment by asvetlov** [comment]
Are we looking on the same logo?
Sure it is spherical!

**Comment by webknjaz** [comment]
I mean, is it weird to me here:
screenshot_2018-02-07-21-05-09_com facebook orca

**Comment by asvetlov** [comment]
Agree.
Let's merge and work on logo in other PR

**Pull request merged**